### PR TITLE
Vdb backfill storage

### PR DESCRIPTION
**Pull request by elizabethengelman**

### CODE REVIEW

File: dockerfiles/backfill_storage/startup_script.sh [hunk 1]
```diff
@@ -28,5 +28,5 @@ if test -z "$VDB_PG_CONNECT"; then
 fi
 
 # Run getStorageValue
-echo "Running backfillStorage from block <NEEDS UPDATE> to <NEEDS UPDATE>"
-./vulcanizedb backfillStorage -s=10739859 -e=10906308 --config config.toml # TODO: update block numbers before next run
+echo "Running backfillStorage from block 109326661 to 10970892"
+./vulcanizedb backfillStorage -s=109326661 -e=109708921 --config config.toml # TODO: update block numbers before next run
```
Comment (rmulhol) [inline dockerfiles/backfill_storage/startup_script.sh]: nbd but it seems like the ending block numbers differ by 1 between the logs and the config

File: environments/backfillEvents.toml [hunk 2]
```diff
@@ -2,250 +2,7 @@
...
-        repository = "github.com/makerdao/vdb-mcd-transformers"
-        migrations = "db/migrations"
-        contracts = [
-            "MCD_FLIP_COMP_A_1.1.2",
-            "MCD_FLIP_LINK_A_1.1.2",
-            "MCD_FLIP_LRC_A_1.1.2",
-        ]
-        rank = "0"
-
+    transformerNames = []
```
Comment (rmulhol) [inline environments/backfillEvents.toml]: 👍

File: dockerfiles/backfill_storage/startup_script.sh (hunk 1 above)
Comment (rmulhol) [reply dockerfiles/backfill_storage/startup_script.sh]: also I know this sorta breaks precedent but I'd be in favor of starting 1 block before the deploy - can't really explain why but it has something to do with guaranteeing that we get all the deployment stuff, and I think it should be super low cost (process is lightweight on blocks without diffs)

**Review by rmulhol — APPROVED** [review]
🚢

File: dockerfiles/backfill_storage/startup_script.sh [hunk 3]
```diff
@@ -28,5 +28,5 @@ if test -z "$VDB_PG_CONNECT"; then
 fi
 
 # Run getStorageValue
-echo "Running backfillStorage from block <NEEDS UPDATE> to <NEEDS UPDATE>"
-./vulcanizedb backfillStorage -s=10739859 -e=10906308 --config config.toml # TODO: update block numbers before next run
+echo "Running backfillStorage from block  10932663 to 10970892"
+./vulcanizedb backfillStorage -s=10932663 -e=109708921 --config config.toml # TODO: update block numbers before next run
```
Comment (elizabethengelman) [inline dockerfiles/backfill_storage/startup_script.sh]: This is the `MCD_FLIP_COMP_A_1.1.2` deploy block - 1

File: dockerfiles/backfill_storage/startup_script.sh [hunk 4]
```diff
@@ -28,5 +28,5 @@ if test -z "$VDB_PG_CONNECT"; then
 fi
 
 # Run getStorageValue
-echo "Running backfillStorage from block <NEEDS UPDATE> to <NEEDS UPDATE>"
-./vulcanizedb backfillStorage -s=10739859 -e=10906308 --config config.toml # TODO: update block numbers before next run
+echo "Running backfillStorage from block  10932663 to 10970892"
+./vulcanizedb backfillStorage -s=10932663 -e=10970892 --config config.toml # TODO: update block numbers before next run
```
Comment (elizabethengelman) [inline dockerfiles/backfill_storage/startup_script.sh:32]: This starting block is the MCD_FLIP_COMP_A_1.1.2 deploy block - 1, and the ending block is the block about when the geth process was restarted with these addresses.

**Pull request merged**